googletest/test/gtest-port_test.cc

```diff
@@ -75,8 +75,8 @@ TEST(IsXDigitTest, WorksForNarrowAscii) {
 }
 
 TEST(IsXDigitTest, ReturnsFalseForNarrowNonAscii) {
-  EXPECT_FALSE(IsXDigit(static_cast<char>(0x80)));
-  EXPECT_FALSE(IsXDigit(static_cast<char>('0' | 0x80)));
+  EXPECT_FALSE(IsXDigit(static_cast<char>(0x80u)));
+  EXPECT_FALSE(IsXDigit(static_cast<char>('0' | 0x80u)));
```

Comment:
I don't see the improvement here.
If 'char' is signed, you've got an impl-defined out of range unsigned-to-signed conversion.
What would work reliably?

static_cast<char>(static_cast<signed char>(-1)) ?

Is there a more concise way to say it?